daml-lf/kv-transaction-support/BUILD.bazel

```
@@ -0,0 +1,43 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_library",
+    "da_scala_test_suite",
+    "lf_scalacopts",
+)
+load("//bazel_tools:scala.bzl", "silencer_plugin")
+
+da_scala_library(
+    name = "kv-transaction-support",
+    srcs = glob(["src/main/**/*.scala"]),
+    scala_deps = [
+        "@maven//:org_scalaz_scalaz_core",
+    ],
+    scalacopts = lf_scalacopts,
+    tags = ["maven_coordinates=com.daml:daml-kv-transaction-support:__VERSION__"],
```

To follow Maven conventions, this should rather be:
```suggestion
    tags = ["maven_coordinates=com.daml:daml-lf-kv-transaction-support:__VERSION__"],
```

---

good catch.